v2/federation/handler.go

```diff
@@ -9,14 +9,34 @@ import (
 	"github.com/spiffe/go-spiffe/v2/spiffeid"
 )
 
-// Handler is an HTTP handler that returns the JSON encoded bundle for the
-// given trust domain the SPIFFE Trust Domain and Bundle specification. The
+type HandlerOption interface {
+	apply(*handlerConfig) error
+}
+
+func WithLogger(log logger.Logger) HandlerOption {
+	return handlerOption(func(c *handlerConfig) error {
+		c.log = log
+		return nil
+	})
+}
+
+// NewHandler returns an HTTP handler that returns the JSON encoded bundle for
+// the given trust domain the SPIFFE Trust Domain and Bundle specification. The
```

Comment:
Looks like this sentence is not well constructed? Missing a comma or something?

Reply:
Thanks @amartinezfayo, fixed.